app/page/info.tsx

```diff
@@ -140,133 +132,89 @@ const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+            <span>{`${userOp.status}: `}</span>
+            <button
+              onClick={handleClick}
+              data-url={`${explorerUrl}userOpHash/${userOp.hash}?network=${chainName}`}>
+              {`${address.ellipsize(userOp.hash)}`}
+            </button>
+          </div>
+        ))
+      )}
+    </div>
   )
 }
 
-const transactionsCrawler = async (type: transactionType, url: string) => {
-  try {
-    const response = await fetch(url)
-    const html = await response.text()
-    const doc = new DOMParser().parseFromString(html, "text/html")
-
-    let transactionXpath = ""
-    let blockXpath = ""
-    let dateXpath = ""
-
-    if (type === transactionType.internal) {
-      transactionXpath =
-        "/html/body/main/section[2]/div[4]/div[2]/div/div[2]/table/tbody/tr/td[1]/div/a"
-      blockXpath =
-        "/html/body/main/section[2]/div[4]/div[2]/div/div[2]/table/tbody/tr/td[2]/a"
-      dateXpath =
-        "/html/body/main/section[2]/div[4]/div[2]/div/div[2]/table/tbody/tr/td[4]/span"
-    }
-    if (type === transactionType.normal) {
-      transactionXpath =
-        "/html/body/main/section[2]/div[4]/div[1]/div/div[2]/table/tbody/tr/td[2]/div/a"
-      blockXpath =
-        "/html/body/main/section[2]/div[4]/div[1]/div/div[2]/table/tbody/tr/td[4]/a"
-      dateXpath =
-        "/html/body/main/section[2]/div[4]/div[1]/div/div[2]/table/tbody/tr/td[6]/span"
-    }
-
-    const transactionNodes = doc.evaluate(
-      transactionXpath,
-      doc,
-      null,
-      XPathResult.ORDERED_NODE_SNAPSHOT_TYPE,
-      null
-    )
-    const blockNodes = doc.evaluate(
-      blockXpath,
-      doc,
-      null,
-      XPathResult.ORDERED_NODE_SNAPSHOT_TYPE,
-      null
-    )
-    const dateNodes = doc.evaluate(
-      dateXpath,
-      doc,
-      null,
-      XPathResult.ORDERED_NODE_SNAPSHOT_TYPE,
-      null
-    )
-
-    const hashes: HexString[] = []
-    const blocksSet: Set<number> = new Set()
-    const dates: Date[] = []
-
-    for (let i = 0; i < transactionNodes.snapshotLength; i++) {
-      const hash = transactionNodes.snapshotItem(i).textContent.trim()
-      const blockNumber = parseInt(
-        blockNodes.snapshotItem(i).textContent.trim(),
-        10
-      )
-      const dateString =
-        (dateNodes.snapshotItem(i) as Element)?.getAttribute("data-bs-title") ||
-        ""
-      if (!blocksSet.has(blockNumber)) {
-        hashes.push(hash)
-        blocksSet.add(isNaN(blockNumber) ? 0 : blockNumber)
-        dates.push(new Date(dateString))
-      }
-    }
-
-    const blocks: number[] = [...blocksSet]
-
-    console.log(`hashes: ${format(hashes)}\nblocks: ${format(blocks)}`)
-
-    return Promise.resolve(hashes)
-  } catch (error) {
-    console.error("Error fetching transaction data:", error)
-    return Promise.reject("Error fetching transaction data")
-  }
-}
-
-const getExplorerUrl = (chain: string | number): string => {
+const getChainName = (chain: string | number): string => {
```

Comment:
I don't think we need to support so many chains listing here. Could you keep only those matter to us?